operator_ui/src/api/v2/ocrKeys.ts

```diff
@@ -0,0 +1,62 @@
+import * as jsonapi from '@chainlink/json-api-client'
+import { boundMethod } from 'autobind-decorator'
+import * as models from 'core/store/models'
+import * as presenters from 'core/store/presenters'
+/**
+ * Create adds validates, saves a new OcrKey.
+ *
+ * @example "<application>/off_chain_reporting_keys"
+ */
+const CREATE_ENDPOINT = '/v2/off_chain_reporting_keys'
+
+/**
+ * Index lists OcrKeys.
+ *
+ * @example "<application>/off_chain_reporting_keys"
+ */
+const INDEX_ENDPOINT = '/v2/off_chain_reporting_keys'
+
+/**
+ * Destroy deletes a OcrKey.
+ *
+ * @example "<application>/off_chain_reporting_keys/:keyId"
+ */
+interface DestroyPathParams {
+  keyId: string
+}
+const DESTROY_ENDPOINT = '/v2/off_chain_reporting_keys/:keyId'
```

Comment:
Could we
```suggestion
const DESTROY_ENDPOINT = `${INDEX_ENDPOINT }/:keyId`
```

Reply:
Same as above. To simplify this i would change `INDEX_ENDPOINT` -> `ENDPOINT`.

Reply:
```
/**
 * Create adds validates, saves a new OcrKey.
 *
 * @example "POST <application>/off_chain_reporting_keys"
 */
const ENDPOINT = '/v2/off_chain_reporting_keys'

/**
 * Index lists OcrKeys.
 *
 * @example "GET <application>/off_chain_reporting_keys"
 */
const INDEX_ENDPOINT = ENDPOINT

/**
 * Destroy deletes a OcrKey.
 *
 * @example "DELETE <application>/off_chain_reporting_keys/:keyId"
 */
interface DestroyPathParams {
  keyId: string
}
const DESTROY_ENDPOINT = `${ENDPOINT}/:keyId`

```

Reply:
I like your suggestions, let's do it.